op_crates/fetch/lib.rs

```diff
@@ -58,9 +61,22 @@ pub fn init(isolate: &mut JsRuntime) {
   }
 }
 
+pub fn init_buildtime(isolate: &mut JsRuntime) {
+  init(isolate, true)
+}
+
+pub fn init_runtime(isolate: &mut JsRuntime) {
+  init(isolate, false)
+}
```

Comment:
I think these methods are superfluous. Maybe just add a docstring comment to init explaining what the boolean flag does.